bundler/spec/runtime/with_unbundled_env_spec.rb

```diff
@@ -72,32 +72,22 @@ def run_bundler_script(env, script)
       bundle_exec_ruby bundled_app("source.rb")
       expect(out).to eq original
     end
-  end
-
-  shared_examples_for "an unbundling helper" do
-    it "should delete BUNDLE_PATH" do
-      create_file("source.rb", <<-RUBY)
-        print #{modified_env}.has_key?('BUNDLE_PATH')
-      RUBY
-      ENV["BUNDLE_PATH"] = "./foo"
-      bundle_exec_ruby bundled_app("source.rb")
-      expect(last_command.stdboth).to include "false"
-    end
 
-    it "should remove '-rbundler/setup' from RUBYOPT" do
+    it "should remove 'bundler/setup' from RUBYOPT" do
       create_file("source.rb", <<-RUBY)
-        print #{modified_env}['RUBYOPT']
+        print Bundler.original_env['RUBYOPT']
       RUBY
-      ENV["RUBYOPT"] = "-W2 -rbundler/setup #{ENV["RUBYOPT"]}"
+      setup_require = "-r#{lib_dir}/bundler/setup"
+      ENV["RUBYOPT"] = "-W2 #{setup_require} #{ENV["RUBYOPT"]}"
```

Comment:
using `setup_require` here attempts to make the `bundler/setup` path more realistic since it's now explicitly in the lib_dir and not unqualified when it's set for real by bundler.